config/dm_config/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+generatorOptions:
+  disableNameSuffixHash: true
```

Comment:
If you're turning off what Kustomize does, should you just define the default dm config map as yaml to begin with?

Reply:
We're using this option in `manager/kustomization.yaml` as well. I had the same thought about just defining it outright in YAML. Not sure what's the best option here.

Reply:
@roehrich-hpe any input on this one since you're the kustomization expert?  😄

Reply:
No, sorry.  I haven't looked at these options.